lib/onfido.rb

```diff
@@ -232,6 +232,8 @@
 require 'onfido/models/sdk_token_response'
 require 'onfido/models/task'
 require 'onfido/models/task_item'
+require 'onfido/models/timeline_file_reference'
+require 'onfido/models/timeline_file_reference1'
```

Comment:
Something looks wrong, need to dig into 🤔

Reply:
It turns out that `timeline_file_reference1` is unused, fixed that in this https://github.com/onfido/onfido-openapi-spec/pull/82